chartfx-dataset/src/main/java/de/gsi/dataset/serializer/spi/iobuffer/DataSetSerialiser.java

```diff
@@ -38,15 +36,13 @@
  *
  * @author rstein
  */
-public class DataSetSerialiser extends DataSetUtilsHelper {
+public class DataSetSerialiser {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'DataSetSerialiser' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper|Constants)'](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5481451)